administrator/language/en-GB/com_scheduler.ini

```
@@ -60,10 +70,13 @@ COM_SCHEDULER_FIELD_OPTION_INTERVAL_MATCH_MINUTES="Minutes"
 COM_SCHEDULER_FIELD_OPTION_INTERVAL_MATCH_MONTHS="Months"
 COM_SCHEDULER_FIELD_TASK_TYPE="Type ID"
 COM_SCHEDULER_FILTER_SEARCH_DESC="Search in task title and note. Prefix with ID: to search for a task ID."
+COM_SCHEDULER_FILTER_SEARCH_HISTORY_DESC="Search in task title. Prefix with ID: to search for a task ID."
```

```suggestion
COM_SCHEDULER_FILTER_SEARCH_HISTORY_DESC="Search in task title. Prefix with ID: to search for a task ID."
```
`history ID` or `log ID` or ???